test/graphql/mutations/media/update_mutation_test.rb

```diff
@@ -0,0 +1,109 @@
+require 'test_helper'
+
+module Mutations
+  module Models
+    class UpdateMutationTest < ActiveSupport::TestCase
+      let(:medium) { FactoryBot.create(:medium) }
+      let(:medium_id) { Foreman::GlobalId.for(medium_id) }
+      let(:os_id) { Foreman::GlobalId.for(FactoryBot.create(:operatingsystem)) }
+      let(:org_id) { Foreman::GlobalId.for(FactoryBot.create(:organization)) }
+      let(:loc_id) { Foreman::GlobalId.for(FactoryBot.create(:location)) }
+      let(:variables) do
+        {
```

Comment:
I think `id` is missing

```suggestion
        {
          id: medium_id,
```